tasks/internal/process-file-updates-task/README.md

```diff
@@ -16,9 +16,6 @@ replacements to a yaml file that already exists. It will attempt to create a Mer
 | tempDir                        | temp dir for cloning and updates                                                                                                                                                         | Yes      | /tmp/$(context.taskRun.uid)/file-updates |
 | internalRequestPipelineRunName | name of the PipelineRun that called this task                                                                                                                                            | No       | -                                        |
 
-## Changes in 0.1.1
```

Comment:
This change contains a fix for the previous change, right? It's not exactly the same, otherwise it would still have the issue. This should really be a new version and it should say that it makes it idempotent again and fixes the issue.

Reply:
Updated